scripts/promptflow-ci/promptflow_ci.py

```diff
@@ -491,12 +267,7 @@ def check_flow_run_status(
         else:
             handled_failures.append(failure_message)
 
-        for flow_run_identifier in submitted_flow_run_identifiers:
-            flow_id, flow_run_id = flow_utils.resolve_flow_run_identifier(
-                flow_run_identifier)
-            flow_run_link = flow_utils.construct_flow_run_link(ux_endpoint, args.subscription_id,
-                                                               args.resource_group, args.workspace_name,
-                                                               experiment_id, flow_id, flow_run_id)
+        for flow_run_id, flow_run_link in submitted_flow_run_ids.items():
             log_debug(
                 f"Flow run link for run {flow_run_id} to Azure Machine Learning Portal: {flow_run_link}")
```

Comment:
Is flow_run_link scrubbed? If yes, we may need to add our doc link here.

Reply:
updated.